src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPannel from './ButtonPanel';
+
+function App() {
+  return (
+    <>
+      <Display />
+      <ButtonPannel />
+    </>
+  );
+}
+
```

Comment:
Kindly use arrow functions to define your components as requested from the last review.